### PR TITLE
adds a visual tell to veils

### PR DESCRIPTION
**Pull request by tattax**
# Document the changes in your pull request

veils are basically thralls but better cause they have no tells DESPITE being almost made passively by the veil mind ability, they can be deconned with mindshields but at that point if you don't run immediately to cargo and order a shitton of them and start mindshielding literally everyone you'll not be able to tell them apart. Veils have significantly less value than thralls since they literally take 1/60 of the time to make but are still considerably better.

Aside from all of this, you can still conceal your sigils if you dress tighly, which should indicate players that you are on the dspawn's side.

# Spriting

<details>
<summary> Sprites</summary>

image

image

image

image

  
e2AVbhYkRo

ME6yUYVf8W

</details>

# Changelog
:cl:  
rscadd: veils get a visible tell that can be hidden with clothing
/:cl:

### CODE REVIEW

**Comment by ynot01** [comment]
I'm for the visual tell

Max HP cap however should've stayed at 30, veils are way too effective at steamrolling and should not be combat capable

**Comment by MajManatee** [comment]
yeah knock their HP back down

File: yogstation/code/game/gamemodes/darkspawn/veil.dm [hunk 1]
```diff
@@ -31,10 +32,12 @@
 	return ..()
 
 /datum/antagonist/veil/apply_innate_effects(mob/living/mob_override)
-	mob_override.maxHealth -= 40
+	veil_sigils = mutable_appearance('yogstation/icons/mob/actions/actions_darkspawn.dmi', "veil_sigils", -UNDER_SUIT_LAYER) //show them sigils
```
Comment (tattax) [inline yogstation/code/game/gamemodes/darkspawn/veil.dm:36]:
```suggestion
	mob_override.maxHealth -= 40
	veil_sigils = mutable_appearance('yogstation/icons/mob/actions/actions_darkspawn.dmi', "veil_sigils", -UNDER_SUIT_LAYER) //show them sigils
```

File: yogstation/code/game/gamemodes/darkspawn/veil.dm [hunk 2]
```diff
@@ -56,9 +59,8 @@
 		Serve the darkspawn above all else. Your former allegiances are now forfeit. Their goal is yours, and yours is theirs.</b>")
 	to_chat(owner, "<i>Use <b>:[MODE_KEY_DARKSPAWN] or .[MODE_KEY_DARKSPAWN]</b> before your messages to speak over the Mindlink. This only works across your current z-level.</i>")
 	to_chat(owner, "<i>Ask for help from your masters or fellows if you're new to this role.</i>")
-	to_chat(owner, span_danger("Your drained will has left you feeble and weak! You will go down with many fewer hits!"))
 	SEND_SOUND(owner.current, sound ('yogstation/sound/ambience/antag/become_veil.ogg', volume = 50))
```
Comment (tattax) [inline yogstation/code/game/gamemodes/darkspawn/veil.dm:65]:
```suggestion
	to_chat(owner, span_danger("Your drained will has left you feeble and weak! You will go down with many fewer hits!"))
	SEND_SOUND(owner.current, sound ('yogstation/sound/ambience/antag/become_veil.ogg', volume = 50))
```

File: yogstation/code/game/gamemodes/darkspawn/veil.dm [hunk 3]
```diff
@@ -31,10 +32,12 @@
 	return ..()
 
 /datum/antagonist/veil/apply_innate_effects(mob/living/mob_override)
-	mob_override.maxHealth -= 40
+	veil_sigils = mutable_appearance('yogstation/icons/mob/actions/actions_darkspawn.dmi', "veil_sigils", -UNDER_SUIT_LAYER) //show them sigils
+	mob_override.add_overlay(veil_sigils)
 
 /datum/antagonist/veil/remove_innate_effects(mob/living/mob_override)
-	mob_override.maxHealth += 40
+	mob_override.cut_overlay(veil_sigils)
```
Comment (tattax) [inline yogstation/code/game/gamemodes/darkspawn/veil.dm:41]:
```suggestion
	mob_override.maxHealth += 40
	mob_override.cut_overlay(veil_sigils)
```

**Review by tattax — COMMENTED** [review]
sure

**Review by ynot01 — CHANGES_REQUESTED** [review]
update changelog

**Comment by tattax** [comment]
done @ynot01

**Review by ynot01 — APPROVED** [review]

File: code/modules/mob/living/carbon/human/examine.dm [hunk 4]
```diff
@@ -367,6 +367,9 @@
 		else
 			msg += "The pair of holes where [t_His] eyes would be seem unnaturally dark and soulless.\n"
 
+	if(!glasses || !wear_suit && mind?.has_antag_datum(ANTAG_DATUM_VEIL))
```
Comment (Bibby0110) [inline code/modules/mob/living/carbon/human/examine.dm]:
```suggestion
	if((!glasses || !wear_suit) && mind?.has_antag_datum(ANTAG_DATUM_VEIL))
```

**Review by Bibby0110 — APPROVED** [review]

**Review by cuackles — CHANGES_REQUESTED** [review]
talking with baiomu about it, we came to the agreement that they are good but the size its to big, also the sprite from the front torso does look a bit like a slash, so maybe change it a bit so it looks a bit more runic like the others or make the others look a bit more like wounds , also u deleted a stray pixel on the paper wizard so thats cool

**Comment by cuackles** [comment]
@tattax u also gotta update the sides so they match with the front

**Comment by tattax** [comment]
@cuackles

**Review by cuackles — APPROVED** [review]

**Review by ThatLing — APPROVED** [review]

**Pull request merged**